docs/_publish/recommended-integration-features.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Recommended Integration Features
```

Comment:
I have this new doc under the "Publish" tab right now - does it make more sense under "Build"?

Reply:
Or the quick start section? As this is a guide to prep for planning integrations for me. We can maybe ask for some more views to see where it fits best statistically. Let me tag @jessveegee and @marinahand first. They contribute to the docs a lot.

Reply:
@michellechu77 I think Build! Since Private integrations would also benefit from this. Especially if we're thinking upmarket, high use integrations that may remain private indefinitely.

Reply:
@marinahand good thought - there are just so many other docs under "Build" already, I was hesitant. I'll move it over.